data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/bulk/JavaClientAccumulatingUncompressedBulkRequestTest.java

```diff
@@ -0,0 +1,222 @@
+/*
```

Comment:
This class is unchanged from the original `JavaClientAccumulatingBulkRequestTest` class